editoast/src/views/temporary_speed_limits.rs

```diff
@@ -0,0 +1,216 @@
+    request_body = TemporarySpeedLimitCreateForm,
+    responses(
+        (status = 201, body = TemporarySpeedLimitCreateResponse, description = "A list of IDs of the created temporary speed limits" ),
+    )
+)]
+async fn create(
+    State(app_state): State<AppState>,
+    Extension(authorizer): AuthorizerExt,
+    Json(TemporarySpeedLimitCreateForm { speed_limits }): Json<TemporarySpeedLimitCreateForm>,
+) -> Result<Json<TemporarySpeedLimitCreateResponse>> {
+    let authorized = authorizer
+        .check_roles([BuiltinRole::InfraWrite].into())
+        .await
+        .map_err(AuthorizationError::AuthError)?;
+    if !authorized {
+        return Err(AuthorizationError::Unauthorized.into());
+    }
+
+    let db_pool = app_state.db_pool_v2.clone();
+    let conn = &mut db_pool.get().await?;
+
+    let speed_limits_changesets = speed_limits
+        .into_iter()
+        .map(|speed_limit| speed_limit.into_temporary_speed_limit_changeset())
+        .collect::<Vec<_>>();
+    let speed_limits: Vec<_> =
+        TemporarySpeedLimit::create_batch(conn, speed_limits_changesets).await?;
+    let ids: Vec<_> = speed_limits
+        .iter()
+        .map(|speed_limit| speed_limit.id)
+        .collect();
+
+    Ok(Json(TemporarySpeedLimitCreateResponse {
+        created_speed_limits_ids: ids,
+    }))
+}
+
+#[cfg(test)]
+mod tests {
+    use crate::Retrieve;
+    use axum::http::StatusCode;
+    use rstest::rstest;
+    use serde_json::json;
+    use uuid::Uuid;
+
+    use crate::{
+        models::temporary_speed_limits::TemporarySpeedLimit,
+        views::{
+            temporary_speed_limits::TemporarySpeedLimitCreateResponse, test_app::TestAppBuilder,
+        },
+    };
+
+    #[rstest]
+    async fn create_ltv_succeeds() {
+        let app = TestAppBuilder::default_app();
+        let pool = app.db_pool();
+
+        let request = app.post("/temporary_speed_limits").json(&json!(
+            {
+                "speed_limits": [
+                {
+                    "start_date_time": "2024-01-01T08:00:00",
+                    "end_date_time": "2024-01-01T09:00:00",
+                    "track_ranges": [],
+                    "speed_limit": 80,
+                    "obj_id": format!("{}", Uuid::new_v4().to_string()),
+                }]
+            }
+        ));
+
+        let response = app
+            .fetch(request)
+            .assert_status(StatusCode::OK)
+            .json_into::<TemporarySpeedLimitCreateResponse>();
+
+        assert_eq!(response.created_speed_limits_ids.len(), 1);
+
+        let speed_limit_id = response
+            .created_speed_limits_ids
+            .first()
+            .expect("Temporary speed limit query did not return any temporary speed limit ids");
+
+        let speed_limit = TemporarySpeedLimit::retrieve(&mut pool.get_ok(), *speed_limit_id)
+            .await
+            .expect("Failed to retrieve temporary speed limit");
+
+        assert!(speed_limit.is_some());
```

Comment:
I would at least assert the `obj_id` is the one we send (rather unique so if we assert that, we're sure it's not some other objects, I don't trust that we can't have data races between our tests since they're not isolated and use the same DB).